.github/workflows/ci-unit-tests-os.yml

```diff
@@ -78,21 +78,15 @@ jobs:
         # TODO: macOS tests were removed for now, see https://github.com/ZcashFoundation/zebra/issues/6824
         os: [ubuntu-latest]
         rust: [stable, beta]
-        features: ["", " --features getblocktemplate-rpcs"]
+        # When vars.EXPERIMENTAL_FEATURES is empty this should become a single matrix item
+        features: ${{ fromJSON(format('["", "--features {0}"]', vars.EXPERIMENTAL_FEATURES)) }}
```

Comment:
Oops, this isn't right either, because we don't want the features if the list is empty.